src/machines/test_help/async_workflow_driver.rs

```diff
@@ -36,23 +38,23 @@ struct TestWfDriverCache {
 
 impl TestWfDriverCache {
     /// Unblock a command by ID
-    fn unblock(&self, id: &str) {
+    fn unblock(&self, id: CommandID) {
         let mut bc = self.blocking_condvar.0.lock();
-        if let Some(t) = bc.issued_commands.remove(id) {
+        if let Some(t) = bc.issued_commands.remove(&id) {
             t.unblocker.send(()).unwrap()
         };
     }
 
     /// Cancel a timer by ID. Timers get some special handling here since they are always
     /// removed from the "lang" side without needing a response from core.
-    fn cancel_timer(&self, id: &str) {
+    fn cancel_timer(&self, id: CommandID) {
```

Comment:
This should probably only accept a &str and then turn it into a `CommandID::Timer`

Reply:
Done